08-compiler-features.tex

```diff
@@ -239,14 +239,16 @@ \section{Completion}
 	\item[typed modules] compilation modules will be cached after a successful compilation and can be reused in later compilation/completions if none of their dependencies have been modified
 \end{description}
 
-You can get precise reading of the times spent by the compiler and how using the compilation server affects it by adding the \ic{--times} command line.
+You can get precise reading of the times spent by the compiler and how using the compilation server affects them by adding \ic{--times} to the command line.
 
 \paragraph{Protocol}
-As the following Haxe/Neko example shows, you can simply connect on the server port and send all commands (one per line) ending with a 0 binary char. You can then read the results.
+As the following Haxe/Neko example shows, you can simply connect on the server port and send all commands (one per line) ending with a 0 binary char. You can, then, read the results.
 
-Macros and other commands can log events which are not errors. From the command line we can see the difference between what is printed on \ic{stdout} and what is print on \ic{stderr}. This is not the case in socket mode. In order to differentiate between the two, log messages (not errors) are prefixed with a \ic{\\x01} character and all newline-characters in the message are replaced by the same \ic{\\x01} character.
-warnings and other messages can also be considered errors, but are not fatal ones. If a fatal error occurred, we will send a single \ic{\\x02} message-line
-Here's some code that will treat connect to the server and handle the protocol details :
+Macros and other commands can log events which are not errors. From the command line, we can see the difference between what is printed on \ic{stdout} and what is print on \ic{stderr}. This is not the case in socket mode. In order to differentiate between the two, log messages (not errors) are prefixed with a \ic{\\x01} character and all newline-characters in the message are replaced by the same \ic{\\x01} character.
+
+Warnings and other messages can also be considered errors, but are not fatal ones. If a fatal error occurred, it will send a single \ic{\\x02} message-line.
+
+Here's some code that will treat connection to the server and handle the protocol details :
```

Comment:
While you're at it, could you remove the extra space before the colon at the end here?